internal/config/config_test.go

```diff
@@ -0,0 +1,69 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/emmaLP/gs-software-onboarding/internal/model"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadConfig(t *testing.T) {
+	envVars := env{
+		"BASE_URL":          "localhost:8000",
+		"DATABASE_USERNAME": "test_username",
+		"DATABASE_PASSWORD": "test_password",
+		"DATABASE_HOST":     "localhost",
+		"DATABASE_PORT":     "30000",
+		"DATABASE_NAME":     "hackernews",
+	}
+	tests := []struct {
+		name        string
+		expected    *model.Configuration
+		filePath    string
+		envVars     env
+		expectedErr string
+	}{
+		{
+			name:     "Successfully load app.env file",
+			filePath: "../..",
```

Comment:
It might be worth creating an example file in the test case rather than relying on something outside of the test. [Example from the shopify-event-ingestion service](https://github.com/gymshark/shopify-event-ingestion/blob/main/internal/config/config_test.go#L28)

Reply:
Done